gretl/build.gradle

```diff
@@ -155,21 +154,22 @@ pluginBundle {
     }
 }
 
-task setupPluginUpload() {
+tasks.register('setupPluginUpload'){
     description = "Converts Gradle plugin publish key and secret from environment variable to system property. (Workaround)"
     doLast {
-	    def key=System.env.gradlePublishKey
-	    def secret = System.env.gradlePublishSecret
-	
-	    if( !key || !secret)
-	    {
-	        throw new RuntimeException("gradlePublishKey and/or gradlePublishSecret are not defined environment variables")
-	    }
-	
-	    System.properties.setProperty("gradle.publish.key", key)
-	    System.properties.setProperty("gradle.publish.secret", secret)
+        def key=System.env.gradlePublishKey
```

Comment:
we might want to introduce some linter for the future, the code style is very inconsistent across the code base. For example here - no spaces around equals sign on one line, and next line already has them. Also curly brackets should be Java style - first one at the end of a line, not at the beginning of the next one

Reply:
probably the most widely used linter for Java: https://github.com/checkstyle/checkstyle

Reply:
@dkubanyi Thanks for the input. There already is an open issue regarding linting: https://github.com/sogis/gretl/issues/143 . We suggested to use SonarLint. But please add checkstyle as a suggestion to it. We can discuss it with @edigonzales . :)